src/components/Add_Location.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {InputGroup, Label, Input} from 'reactstrap';
+
+const AddLocation = ({ idx, cabRoute, handleRouteChange }) => {
+  const locationId = `location-${idx}`;
+  const creditId = `credit-${idx}`;
+
+   return(
+     <div className="row">
+      <div className="col-sm-6" align="left" key={`location-${idx}`}>
+       <Label style={{ width:"120px" }} for="enterLocation">{`Location No : ${idx + 1}`}</Label>
```

Comment:
Avoid inline styling if its not dynamic

Reply:
I have Done The changes, Please Review It.